README.md

```diff
@@ -71,6 +71,13 @@ The following configuration options are available:
   * Default: 1000000
   * Importance: medium
   
+``large.message.use.headers``
```

Comment:
Currently, the description (besides the configuration documentation) of this SerDe can be found on Medium. In the blog post, the byte flag is described. Maybe the blogpost could be updated, so the header option also occurs there or the Readme here could have some more information about the purpose and how it works :)